BUILD.gn

```diff
@@ -11,6 +11,12 @@ group("flutter") {
     "//flutter/sky",
   ]
 
+  if (current_toolchain == host_toolchain) {
+    deps += [
+        "//dart:create_sdk"
```

Comment:
Do you want to do this for Fuchsia?  I suspect you want to put this in the non-Fuchsia section.